ada_feeding/ada_feeding/behaviors/moveit2/servo_move.py

```diff
@@ -42,17 +42,20 @@ def blackboard_inputs(
         pub_topic: Union[BlackboardKey, str] = "~/servo_twist_cmds",
         pub_qos: Union[BlackboardKey, QoSProfile] = QoSProfile(depth=1),
         default_frame_id: Union[BlackboardKey, str] = "world",
+        curr_distance: Optional[BlackboardKey] = None,  # float
     ) -> None:
         """
         Blackboard Inputs
 
         Parameters
         ----------
         twist: Twist or TwistStamped to publish, with updated time stamps.
-        duration: How long to publish the twist
+        duration: How long to publish the twist. If negative, then run forever.
         pub_topic: Where to publish servo TwistStamped messages
         pub_qos: QoS for publisher
         default_frame_id: frame_id to use if Twist type is provided.
+        curr_distance: If None, use time as distance. Otherwise, this
```

Comment:
This doesn't appear to be used at all in this Behavior. I assume that it is for the MoveItVisitor to calculate time left to move.

If so, I would completely remove it from this behavior. It just makes the class more confusing.

Same with initial_distance.

See my comment in MoveTo Visitor as an idea on how to fix the logic.